tests/acceptance/features/apiSharingNgDriveInvitation/driveShareInvitations.feature

```diff
@@ -962,6 +935,7 @@ Feature: Send a drive invitations
 
 
   Scenario Outline: try to invite user to personal drive with different roles using root endpoint
+    Given the administrator has enabled the permissions role "Secure Viewer"
```

Comment:
If the role is anything other than Secure Viewer, it unnecessarily increases the load on our OCIS server every time ociswrapper needs to stop and restart OCIS. Currently, the share pipeline is already quite long and time-consuming, so we should avoid this scenario when sharing.
```suggestion
```